src/mpid/ch4/src/ch4r_symheap.h

```diff
@@ -312,12 +312,11 @@ static inline int MPIDI_CH4R_get_shm_symheap(MPI_Aint shm_size, MPI_Aint * shm_o
 {
     int mpi_errno = MPI_SUCCESS;
     int iter = MPIR_CVAR_CH4_SHM_SYMHEAP_RETRY;
+    unsigned any_mapfail_flag = 1;
```

Comment:
This fix ("ch4/rma: fix symheap failure checking") should be squashed with an appropriate patch before merging.

Reply:
will squashed